### PR TITLE
Build script for lego golang package

### PR DESCRIPTION
**Pull request by ezeeyahoo**
Hi

Please review and provide your feeback.

Thanks

### CODE REVIEW

File: l/lego/lego_ubi8.3.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    echo "Create the skeleton for your local users go directory"
+    mkdir -p go/{bin,pkg,src}
+
+    echo "Setting up GOPATH"
+    echo "export GOPATH=$(pwd)/go" >> .profile && source .profile
+
+    echo "Setting PATH to include golang binaries"
+    echo "export PATH='$PATH':$(pwd)/golang/go/bin:$GOPATH/bin" >> .profile && source .profile
+
```
Comment (amitsadaphule) [inline l/lego/lego_ubi8.3.sh:66]: I think there's no need to add things to `.profile` etc and complicate the `go` installation steps. Please refer to [this](https://github.com/ppc64le/build-scripts/commit/49ae79e15cd863475fb705a204293982fe3e95bb#diff-4ba2744d2c13d2a179b0ee4edbb7a701da7697a167fd13f1ff8c9084d4e44353R23-R29)

File: l/lego/lego_ubi8.3.sh [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+fi
+
+# install lego package
+GO111MODULE="off" go get github.com/go-acme/lego
+cd $GOPATH/src/github.com/go-acme/lego
+go get -tags $TAGS -t ./...
+make test && make build
+[ $? != 0 ] && exit 1
```
Comment (amitsadaphule) [inline l/lego/lego_ubi8.3.sh:74]: `set -eu` can be used at the beginning of the script to get rid of these command status checks.

**Review by amitsadaphule — CHANGES_REQUESTED** [review]

File: l/lego/lego_ubi8.3.sh [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+# ----------------------------------------------------------------------------
+#
+# Package        : Lego
+# Version        : 3.5.0 - latest
+# Source repo    : https://github.com/go-acme/lego
+# Tested on      : Ubi 8.3 + Go latest version (v1.7.1)
+# Script License : Apache 2.0
+# Maintainer     : Eshant Gupta <eshant.gupta@ibm.com>; Priya Seth <sethp@us.ibm.com>
+#
+# Disclaimer: This script has been tested both in root and user mode on given
```
Comment (amitsadaphule) [inline l/lego/lego_ubi8.3.sh:11]: To keep it simple, we can simply have this script run as root user. Being a _go_ package, it's not necessary to have a capability to run as non-root.

File: l/lego/lego_ubi8.3.sh (hunk 1 above)
Comment (amitsadaphule) [reply l/lego/lego_ubi8.3.sh:73]: We can simply get rid of these writes to .profile/.bashrc/... We're running the script in a UBI container and it is ok to assume that the script will be run in a fresh container.

**Comment by Siddhesh-Ghadi** [comment]
This PR can be closed in favor of #1221.

**Pull request closed without merging**